README.md

```diff
@@ -1,21 +1,261 @@
 # Google Cloud Profiler
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/GoogleCloudPlatform/cloud-profiler-nodejs.svg)](https://greenkeeper.io/)
-
+[![NPM Version][npm-image]][npm-url]
 [![Dependency Status][david-image]][david-url]
 [![devDependency Status][david-dev-image]][david-dev-url]
+[![Known Vulnerabilities][snyk-image]][snyk-url]
+
+> **Alpha**. *This is an Alpha release of Stackdriver Profiler Node.js 
+profiling agent. This feature might be changed in backward-incompatible ways 
+and is not recommended for production use. It is not subject to any SLA or 
+deprecation policy.*
+
+
+## Prerequisites
+
+1. Your application will need to be using Node.js version 6.12.3 or greater, 
+or Node.js 8.9.4 or greater.
```

Comment:
It would be good to add this version requirement to the `engines.node` field in the `package.json` so that users get warned.